applications/dashboard/src/scripts/compatibilityStyles/discussionStyles.ts

```diff
@@ -96,4 +106,55 @@ export const discussionCSS = () => {
         margin: 0,
         padding: 0,
     });
+
+    cssOut(
+        `
+        .Options a.Bookmark,
+        .Options a.Bookmarking,
+        .Options a.Bookmarked
+        `,
+        {
+            opacity: 1,
+            width: unit(24),
+            height: unit(24),
+            display: "block",
+            position: "relative",
+        },
+    );
+
+    cssOut(
+        `
+        .Content a.Bookmark,
+        .Content a.Bookmarking,
+        .Content a.Bookmarked`,
+        {
+            cursor: "pointer",
+        },
+    );
+
+    cssOut(
+        `
+        .Content a.Bookmark .svgBookmark,
+        .Content a.Bookmarking .svgBookmark,
+        .Content a.Bookmarked .svgBookmark`,
+        {
+            ...absolutePosition.topLeft("50%", "50%"),
+            content: quote(``),
+            display: "block",
+            width: unit(12),
+            height: unit(16),
+            fontSize: unit(12),
+            transform: translate(`-50%`, `-50%`),
+        },
+    );
+
+    cssOut(
+        `
+        body.Discussions .DataList .Options,
+        body.Discussions .MessageList .Options
+    `,
```

Comment:
same question?
Shouldn't we either do it all the time when start and when end the string, or do never.
Just to have it styled the same way across the project